### PR TITLE
2 Updates for M46 post and also updating associated docs about manifest

### PR DESCRIPTION
**Pull request by PaulKinlan**
@jpmedley sorry for two posts in one, originally it was one post but I want to isolate them out.

In the same set of commits I have also update some of the articles as well around the manifest.

### CODE REVIEW

**Comment by jpmedley** [comment]
Reviewing now.

File: src/_langs/en/updates/2015-08-29-using-manifest-to-set-sitewide-theme-color.markdown [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+"theme_color": "#2196F3"
+{% endhighlight %}
+
+To see this in action, visit <a href="https://airhorner.com">Airhorner &mdash; the worlds best airhorn</a> 
+and add it to your home screen. Or look at the <a href="https://airhorner.com/manifest.json">site's manifest</a>.
+
+
+### FAQ
+
+* <strong>Does this apply if my site is not launched from the homescreen?</strong>
```
Comment (arthurvr) [inline src/_langs/en/updates/2015-08-29-using-manifest-to-set-sitewide-theme-color.markdown]: Maybe use actual markdown instead of HTML `<strong>` tags?

File: src/_langs/en/fundamentals/device-access/stickyness/web-app-manifest.markdown [hunk 2]
```diff
@@ -136,9 +144,88 @@ Don't worry, if you think users would prefer to view your page as a normal site
 
 <div class="clear"></div>
 
+## Create a splash screen
+
+When you launch your web app from the homescreen a number of things happen behind the
+scenes:
+
+1. Chrome needs to launch,
```
Comment (jpmedley) [inline src/_langs/en/fundamentals/device-access/stickyness/web-app-manifest.markdown]: End with a period.

File: src/_langs/en/fundamentals/device-access/stickyness/web-app-manifest.markdown [hunk 3]
```diff
@@ -136,9 +144,88 @@ Don't worry, if you think users would prefer to view your page as a normal site
...
+2. The renderer that displays the page needs to start up.
+3. Your site is loaded from the network (or cache like ServiceWorker).
+
+While this is happening the screen will be white to the user and will look like it has stalled.
```
Comment (jpmedley) [inline src/_langs/en/fundamentals/device-access/stickyness/web-app-manifest.markdown]: Delete 'to the user'.

File: src/_langs/en/fundamentals/device-access/stickyness/web-app-manifest.markdown [hunk 4]
```diff
@@ -136,9 +144,88 @@ Don't worry, if you think users would prefer to view your page as a normal site
...
+This becomes especially apparent if you are loading your web page from the network where
+frequently many pages take more than 1 or 2 seconds to get any content visible on the homepage.
+
+To provide a better user experience you can control the color of the screen by adding a
+`background_color` to your manifest and giving it an HTML color value. The color will be used
+by Chrome the instant the web app is launched and will remain on the screen until the web
+apps first render.
```
Comment (jpmedley) [inline src/_langs/en/fundamentals/device-access/stickyness/web-app-manifest.markdown]: The word apps needs an apostrophe.

File: src/_langs/en/fundamentals/device-access/stickyness/web-app-manifest.markdown [hunk 5]
```diff
@@ -136,9 +144,88 @@ Don't worry, if you think users would prefer to view your page as a normal site
...
+
+Simply set the following in your manifest.
```
Comment (jpmedley) [inline src/_langs/en/fundamentals/device-access/stickyness/web-app-manifest.markdown]: You could just shorten this to 'For example:'. Otherwise this is redundant with lines 160/161.

File: src/_langs/en/updates/2015-08-29-using-manifest-to-set-sitewide-theme-color.markdown (hunk 1 above)
Comment (PaulKinlan) [inline src/_langs/en/updates/2015-08-29-using-manifest-to-set-sitewide-theme-color.markdown]: The parser doesn;t always seems to like bullet point followed by either _ or *

Comment (arthurvr) [inline src/_langs/en/updates/2015-08-29-using-manifest-to-set-sitewide-theme-color.markdown]: :+1: It's valid markdown though, should probably report an upstream bug.

File: src/_langs/en/fundamentals/device-access/stickyness/web-app-manifest.markdown [hunk 6]
```diff
@@ -136,9 +144,88 @@ Don't worry, if you think users would prefer to view your page as a normal site
...
+
+        <figcaption>Sitewide theme color</figcaption>
+    </figure>
+</div>
+
 ## Define the initial orientation of the page
 
-You can enforce a specific orientation, which is really useful for some use cases like games, which may only work in landscape. However, this should be used with care. Users prefer being able to view apps in both orientations.
+You can enforce a specific orientation, which is really useful for some use cases like games,
+which may only work in landscape. However, this should be used with care. Users prefer being
```
Comment (jpmedley) [inline src/_langs/en/fundamentals/device-access/stickyness/web-app-manifest.markdown]: Suggest change to 'which may only work in one orientation. Use this with care. Users prefer selecting the orientation.'

File: src/_langs/en/updates/2015-08-28-using-web-app-manifest-to-set-solid-color-loading-screen.markdown [hunk 7]
```diff
@@ -0,0 +1,74 @@
+---
```
Comment (jpmedley) [inline src/_langs/en/updates/2015-08-28-using-web-app-manifest-to-set-solid-color-loading-screen.markdown]: This looks like a duplicate of the content above. Please copy whatever edits you make.

**Comment by jpmedley** [comment]
@PaulKinlan Just a few things.

**Comment by PaulKinlan** [comment]
@jpmedley cool, will sort.  and next time will do it on a local branch :)
